package.json

```diff
@@ -21,5 +21,10 @@
   "bugs": {
     "url": "https://github.com/Sergey60327/liri-node-app/issues"
   },
-  "homepage": "https://github.com/Sergey60327/liri-node-app#readme"
+  "homepage": "https://github.com/Sergey60327/liri-node-app#readme",
+  "dependencies": {
```

Comment:
You need to make sure that you save your dependencies to the `package.json` so when other people clone your repo they can simply run `npm install` to get all the necessary external packages. To save a package to the `package.json` you just need to pass the `--save` flag. If you were trying to save the spotify package for instance, you would just run `npm install --save spotify`